### PR TITLE
Ecomm recsys

### PR DESCRIPTION
**Pull request by robertdhayanturner**
Ecomm Recsys notebook article

### CODE REVIEW

File: docs/articles/ecomm-recys.md [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+## Building an e-commerce recommendation engine with Superlinked
+
+At the start of development, we have the following product data:
+- number of reviewers
+- product ratings
+- textual description
+- name of product (usually containing brand name)
+- category
+  
+We have two users (user_1 and user_2) whom we can differentiate by either:
```
Comment (robertdhayanturner) [inline docs/articles/ecomm-recys.md:33]: @morkapronczay
differentiate them by 1 specific on-registration product pref
2 seems like there are two points here: A. in general, users prefer cheaper, more reviewed, higher rated products (so this isn't a way of differentiating them), B. users prefer certain kinds of products (this isn't a way of differentiating user_1 and user_2).

Comment (morkapronczay) [reply docs/articles/ecomm-recys.md:33]: yes agreed, better formulation!

File: docs/articles/ecomm-recys.md [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+- which of three products offered during registration they chose, or
+- more general characteristics explained in the below paragraph (price, reviews)
+ 
+Users have preferences in textual characteristics of products (description, name, category), and (according to classical economics) ceteris paribus prefer products that:
+- cost less
+- have a lot of reviews
+- have higher ratings
+
```
Comment (robertdhayanturner) [inline docs/articles/ecomm-recys.md:41]: @morkapronczay Do you mean we *know* this from user behavior (event data), or we *assume* this based on classical economics?

Comment (robertdhayanturner) [reply docs/articles/ecomm-recys.md:40]: @morkapronczay see above question.

Comment (robertdhayanturner) [reply docs/articles/ecomm-recys.md:40]: @morkapronczay see above question.

Comment (morkapronczay) [reply docs/articles/ecomm-recys.md:40]: yeah common sense or classical economics :)

File: docs/articles/ecomm-recys.md [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+- number of reviewers
+- product ratings
+- textual description
+- name of product (usually containing brand name)
+- category
+  
+We also have the following **data about users and products**:
+- 1. each user chooses one of three products offered when they register (i.e., product preference data)
+- 2. user behavior (*after* registration) provides additional event data - preferences for textual characteristics of products (description, name, category)
+- 3. also, classical economics tells us that, on average, all users ceteris parabis prefer products that:
```
Comment (morkapronczay) [inline docs/articles/ecomm-recys.md:36]: this point might work better separated as you have pointed out this is not exactly data about the users, more like learned insights

**Review by morkapronczay — APPROVED** [review]

File: docs/articles/ecomm-recys.md (hunk 3 above)
Comment (robertdhayanturner) [reply docs/articles/ecomm-recys.md]: agree.

**Pull request merged**